docs/architecture/cloud-native/azure-data-storage.md

```diff
@@ -173,7 +173,7 @@ You manage data in Cosmos DB data by creating [databases, containers, and items]
 
 **Figure 5-17**: Hierarchy of Cosmos DB entities
 
-Note in Figure 5-17 how you start by creating a Cosmos DB database inside of an Azure account. That database becomes the unit of management for a set of containers. A container is a schema-agnostic grouping of items that can be expressed as a collection, table, or graph, based on your selected API provider (discussed in the prior section). Items are the data that you add to the container and are represented as documents, rows, nodes, or edges. By default, all items that you add to a container are automatically indexed without requiring explicit index or schema management.
+Note in Figure 5-17 how you start by creating a Cosmos DB database inside of a database account. That database becomes the unit of management for a set of containers. A container is a schema-agnostic grouping of items that can be expressed as a collection, table, or graph, based on your selected API provider (discussed in the prior section). Items are the data that you add to the container and are represented as documents, rows, nodes, or edges. By default, all items that you add to a container are automatically indexed without requiring explicit index or schema management.
```

Comment:
The image shows the databases under "Database Accounts", not "Azure account"

Reply:
Great catch! Agree with the change. But, I will add it in my upcoming Pull Request for this chapter. That way, we only do a single merge.